spring-bitcoin-fee/spring-bitcoin-fee-autoconfigure/src/main/java/org/tbk/bitcoin/tool/fee/dummy/config/DummyFeeProviderAutoConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package org.tbk.bitcoin.tool.fee.dummy.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.tbk.bitcoin.tool.fee.dummy.ConfigurableDummyFeeSource;
+import org.tbk.bitcoin.tool.fee.dummy.DummyFeeProvider;
+import org.tbk.bitcoin.tool.fee.dummy.DummyFeeSource;
+
+import static java.util.Objects.requireNonNull;
+
+@Configuration
+@EnableConfigurationProperties(DummyFeeProviderAutoConfigProperties.class)
+@ConditionalOnProperty(name = "org.tbk.bitcoin.tool.fee.dummy.enabled", havingValue = "true")
```

Comment:
Missing `@ConditionalOnClass` annotation here.

Without this a "ClassNotFound" may occurs if property set but module not implemented.